static/js/components/SkipFinancialAidDialog.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import Button from 'react-mdl/lib/Button';
+
+const skipActions = (cancel, skip) => (
+  <div className="actions">
+    <Button
+      type='button'
+      className="dialog-button cancel-button"
+      onClick={cancel}>
+      Cancel
+    </Button>
+    <Button
+      type='button'
+      className="dialog-button save-button"
+      onClick={skip}>
+      Pay Full Price
```

Comment:
haha, mine is too! didn't notice. I think this is because those buttons have a width set on them.